app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= render 'shared/flashes' %>
+<div class="container-b">
```

Comment:
- [ ] Kindly follow the given wire frame for user index page

Wireframe:
![image](https://user-images.githubusercontent.com/62705947/219148220-82cf4698-44a5-4306-9ed9-b389422d3e2d.png)
Yours:
![image](https://user-images.githubusercontent.com/62705947/219148411-01ce5740-c4d6-4025-9a32-0ad68968137b.png)

Reply:
- [ ] For user show page please follow the given wireframe
![image](https://user-images.githubusercontent.com/62705947/219148714-5a7a84d7-d101-4c21-a2a9-565896035511.png)
![image](https://user-images.githubusercontent.com/62705947/219148785-925199da-bf07-4cd5-85b9-d2e08a18f2bb.png)

Reply:
fixed